packages/cli/src/rollup/config.ts

```diff
@@ -0,0 +1,201 @@
+
+  // Check if the indexers directory exists and is not empty
+  let indexerFiles: string[] = [];
+  try {
+    indexerFiles = fsExtra
+      .readdirSync(indexerDir)
+      .filter((file) => file.endsWith(".indexer.ts"));
+    if (indexerFiles.length === 0) {
+      console.warn(`No indexer files found in ${indexerDir}`);
+    }
+  } catch (error) {
+    console.error(`Error reading indexers directory: ${error}`);
+  }
+
+  const indexerImports = indexerFiles
+    .map(
+      (file, index) =>
+        `import indexer${index} from '${join(indexerDir, file)}';`,
+    )
+    .join("\n");
+
+  // Generate main.ts content
+  const mainContent = `
+import { createClient } from "@apibara/protocol";
+import { createIndexer, run } from "@apibara/indexer";
+import consola from "consola";
+import { defineCommand, runMain } from "citty";
+import config from './${configPath}';
+
+${indexerImports}
+
+const indexers = {
+  ${indexerFiles
+    .map(
+      (file, index) => `'${basename(file, ".indexer.ts")}': indexer${index},`,
+    )
+    .join("\n  ")}
+};
+
+const command = defineCommand({
+  meta: {
+    name: "run-indexers",
+    description: "Run Apibara indexers",
+  },
+  args: {
+    indexers: {
+      type: "string",
+      description: "Comma-separated list of indexers to run",
+    },
+    preset: {
+      type: "string",
+      description: "Preset to use",
+    },
+    sink: {
+      type: "string",
+      description: "Sink to use",
+    },
+  },
+  async run({ args }) {
+    const selectedIndexers = args.indexers ? args.indexers.split(',') : Object.keys(indexers);
+    const preset = args.preset || config.preset || 'default';
+    const sinkName = args.sink || 'default';
+
+    // Apply preset
+    let runtimeConfig = { ...config.runtimeConfig };
+    if (preset && config.presets && config.presets[preset]) {
+      runtimeConfig = { ...runtimeConfig, ...config.presets[preset].runtimeConfig };
+    }
+
+    // Get sink function
+    const sinkFunction = config.sink?.[sinkName];
+    if (!sinkFunction) {
+      throw new Error(\`Sink \${sinkName} not found\`);
+    }
+
+    await Promise.all(selectedIndexers.map(async (name) => {
+      if (!indexers[name]) {
+        console.error(\`Indexer \${name} not found\`);
+        return;
+      }
+
+      const indexerConfig = indexers[name]
+      const indexer = typeof indexerConfig === 'function'
+        ? await createIndexer(indexerConfig(runtimeConfig))
+        : createIndexer(indexerConfig);
+
+      const client = createClient(indexer.streamConfig, indexer.options.streamUrl);
+      const sink = sinkFunction();
+
+      try {
+      consola.log("Running Indexer: ", name);
+        await run(client, indexer, sink);
+      } catch (error) {
+        consola.error(\`Error in indexer \${name}:\`, error);
+      }
+    }));
+  },
+});
+
+runMain(command);
+`;
+
+  return {
+    input: {
+      main: "virtual:main.ts",
+    },
+    output: {
+      dir: join(apibara.options.outputDir || "dist"),
```

Comment:
IMHO the cli should output the build files to `.apibara/build`, that way a project can be cleaned up by deleting just one folder.